dolphinscheduler-master/src/test/java/org/apache/dolphinscheduler/server/master/registry/ServerNodeManagerTest.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.dolphinscheduler.server.master.registry;
```

Comment:
please add license header

Reply:
Thank you for reminding, the License Header has been added